src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
+import Calculator from './components/calculator';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Please change this component to class bases component as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/212910768-dd76c6a6-a1cd-440c-b6b4-42e17ad1c48a.png)